cli/src/main.rs

```diff
@@ -37,10 +37,11 @@ fn main() {
         }
 
         println!(
-            "OS information:\nType: {}\nVersion: {}\nBitness: {}",
+            "OS information:\nType: {}\nVersion: {}\nBitness: {}\nFamily: {}",
             info.os_type(),
             info.version(),
-            info.bitness()
+            info.bitness(),
+            info.family(),
```

Comment:
Do you think it is useful to show the family here? I'm a little afraid for the output to become too verbose. Additionally it can be a little weird in some cases. For example, currently it will be "OS Windows, Family: Unknown".

Reply:
I can remove that out. Probably not so useful on output, but I also assumed the output here was probably mostly for eyes on validation since os_info's biggest use case is probably as a library to gather the data needed and the caller would just grab the info and the fields it needs.

Reply:
> Mhmm, that is odd. I got that output from my work machine which is Windows 11, but shows the version as Windows 10.

According to [this information](https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/wdm/ns-wdm-_osversioninfoexw#remarks) it is correct, but I agree that it can be confusing. Ideally, it should be something like "Windows 11 (10.0.22000)".

Reply:
Interesting, yea that is confusing. You would think you see "10" as a major number for versioning, it specifically would reference Windows 10 since they changed from their random naming scheme of XP, Vista, Millennium to 7, 8, 10, 11.